### ISSUE TITLE
user_token and secret for trello

### ISSUE THREAD

**Issue by xstable** [cprice404/redmine-trello#14]
I'm not sure, if this Project is outdated?!
I use Redmine 2.4 and try to setup your plugin.
I found the app_key on the Page: https://trello.com/1/appKey/generate

But there is no way to create or see a user-token. 
Could you give me a hint, what to do?

**Comment by cprice404** [comment]
@xstable yeah, sorry, we haven't used this in a couple of years now.  It's possible that trello changed their auth model.  If you figure it out and want to file a PR updating the links in that sample config file I'll be happy to merge it!
